src/config.c

```diff
@@ -3206,6 +3206,7 @@ standardConfig static_configs[] = {
     createBoolConfig("cluster-slot-stats-enabled", NULL, MODIFIABLE_CONFIG, server.cluster_slot_stats_enabled, 0, NULL, NULL),
     createBoolConfig("hide-user-data-from-log", NULL, MODIFIABLE_CONFIG, server.hide_user_data_from_log, 1, NULL, NULL),
     createBoolConfig("import-mode", NULL, DEBUG_CONFIG | MODIFIABLE_CONFIG, server.import_mode, 0, NULL, NULL),
+    createBoolConfig("allow-empty-set", NULL, MODIFIABLE_CONFIG, server.allow_empty_set, 0, NULL, NULL),
```

Comment:
I don't think we want this as a configuration. Application developers should be the ones that get to decided what behavior they want related to empty sets.

Reply:
Thanks for the comment.
I considered introducing new commands, but IMO, there are potential downsides:
1. New commands like SREMMT, SMOVEMT, and SPOPMT could increase the complexity of command management. This might lead to further requests to implement similar "MT" commands for other data types, such as ```lists``` and ```sorted sets```, adding more maintenance overhead.
2. Using a configuration setting is more flexible and could be extended to support other data types in the future like ```allow-empty-object```, making it a more scalable solution. 

Could I get thoughts on this?